src/components/QuizGame/index.jsx

```diff
@@ -34,16 +35,16 @@ const QuizGame = () => {
 
   const checkAnswer = (e) => {
     if (isGameStart) {
-      const userChosenAnswer = e.currentTarget.value;
+      const solutionAnswer = e.currentTarget.value;
```

Comment:
you changed the wrong one. this is the userChosenAnswer, which was correct. reread the comment the line is on/reread your own code

Reply:
PTAL